content/articles/okta-identity-provider.md

```diff
@@ -89,6 +76,7 @@ If the Okta app is no longer linked (e.g. access token is revoked or event hook
 To give people access to DNSimple, click "Assignments" under the configured DNSimple app in your Okta dashboard. Click "Assign", and select the appropriate users.
 
 ![Assign users and groups](/files/okta-assignment.png)
```

Comment:
We need a new screenshot here — this is our old UX, but I can't update the picture since I don't have access to an Okta account.